src/cls/rgw/cls_rgw_types.h

```diff
@@ -1045,7 +1045,7 @@ WRITE_CLASS_ENCODER(cls_rgw_gc_obj_info)
 
 struct cls_rgw_lc_obj_head
 {
-  time_t start_date;
+  time_t start_date = 0;
```

Comment:
It depends on what date should be the `start_date`. If it is the system's current date, then initialize with `time(0);`. Please verify it.

Reply:
@tchaikov @joscollin Done Changes Thanks.